packages/crew_launcher.rb

```diff
@@ -0,0 +1,31 @@
+require 'package'
+
+class Crew_launcher < Package
+  description 'Add Chromebrew app to launcher'
+  homepage 'https://github.com/skycocker/chromebrew'
+  version '1'
+  license 'GPL-3'
+  compatibility 'all'
+  source_url 'https://github.com/supechicken/crew-launcher.git'
+  git_hashtag 'main'
+  
+  depends_on 'graphicsmagick'
+
+  def self.install
+    FileUtils.mkdir_p [
+      "#{CREW_DEST_PREFIX}/bin/",
+      "#{CREW_DEST_PREFIX}/lib/crew-launcher/",
+      "#{CREW_DEST_PREFIX}/share/crew-launcher/",
+      "#{CREW_DEST_DIR}/tmp/crew-launcher/",
+      "#{CREW_DEST_PREFIX}/etc/env.d/"
+    ]
+
+    FileUtils.cp_r Dir['*'], "#{CREW_DEST_PREFIX}/lib/crew-launcher/"
+    FileUtils.ln_s "#{CREW_LIB_PATH}/lib/color.rb", "#{CREW_DEST_PREFIX}/lib/crew-launcher/lib"
```

Comment:
Rather than `#{CREW_DEST_PREFIX}/lib`, I think a better location would be `#{CREW_DEST_PREFIX}/share`.  You have it in the array of directories above but yet it's not being used.

Reply:
Only the scripts will save in `#{CREW_DEST_PREFIX}/lib`, the profiles/converted icons will save in `#{CREW_DEST_PREFIX}/share`

Those directories are created for the main script (like a placeholder)

Reply:
It's fine to add both in `#{CREW_DEST_PREFIX}/share` to isolate all the package files in a single location.  Should we add a `def self.remove` section and prompt to remove profiles?

Reply:
Moved all files to `#{CREW_DEST_PREFIX}/share`